client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -1,65 +1,75 @@
 /**
- * External depe;ndencies
+ * External dependencies
  */
 import React from 'react';
+
 /**
  * Internal dependencies
  */
 import ProgressBar from 'components/progress-bar';
 import Rating from 'components/rating';
 import analytics from 'lib/analytics';
 
-/**
- * Constants
- */
-const REVIEW_URL = 'https://wordpress.org/support/view/plugin-reviews/';
-
-module.exports = React.createClass( {
+export default React.createClass( {
```

Comment:
It may be beyond the scope of this PR, but we could use an ES6 class here too:

```js
import React, { Component } from 'react';
// ...
class PluginRatings extends Component {
  // ...
  render() {
    const { translate } = this.props;
    // ...
  }
}

export default localize( PluginRatings ); // Injects `translate` as a prop
```